### PR TITLE
[PGPNP] Update configs/readme for distributed partners to run signer service

### PR DESCRIPTION
**Pull request by aslawson**
### Description

- Update configs/readme for distributed partners to run signer service in preparation for publishing service
- TODO -- decide if threshold values should be configurable by distributed partners in the .env or not

### Other changes

NA

### Tested

Made sure services still run and function locally.

### Related issues

https://github.com/celo-org/celo-monorepo/issues/3899

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/phone-number-privacy/signer/README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+- `BLOCKCHAIN_PROVIDER` - The blockchain node provider for on chain requests. This could be a node with RPC set up.  For development testing you can use https://alfajores-forno.celo-testnet.org
+- `DB_HOST` - The URL under which your database is accessible.  Supports Postgres, MSSQL, MySQL, MariaDB, SQLite3, Oracle.
+- `DB_USERNAME` - DB configuration: The DB username.
+- `DB_PASSWORD` - DB configuration: The DB password.
+- `DB_DATABASE` DB configuration: The DB database name.
+
+Currently, the service only supports a key maintained in Azure Key Vault (AKV).  You can set the following environment variables to configure the AKV connection.
+
+- `KEYVAULT_AZURE_CLIENT_ID` - Azure KeyVault configuration: Client id.
+- `KEYVAULT_AZURE_CLIENT_SECRET` - Azure KeyVault configuration: Client secret.
```
Comment (codyborn) [inline packages/phone-number-privacy/signer/README.md]: These need to be from a service principal account that they create. We can link instructions here:
https://www.npmjs.com/package/@azure/keyvault-keys#configuring-your-key-vault

File: packages/phone-number-privacy/signer/README.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+- `KEYVAULT_AZURE_TENANT` - Azure KeyVault configuration: Tenant.
+- `KEYVAULT_AZURE_VAULT_NAME` - Azure KeyVault configuration: Vault name.
+- `KEYVAULT_AZURE_SECRET_NAME` - Azure KeyVault configuration: Secret name.
```
Comment (codyborn) [inline packages/phone-number-privacy/signer/README.md]: We should add more to these descriptions:

- ClientId of the service principal account that has [Get, List] access to secrets.
- Client Secret of the same service principal account.
- Tenant that the service principal is a member of.
- Name of your Azure Key Vault.
- Name of the secret that holds your BLS key.

**Review by codyborn — APPROVED** [review]

File: packages/phone-number-privacy/signer/README.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+
+## Operations
+
+- `yarn db:migrate` - run to migrate the DB tables.
+- `yarn start` - run to start the service
+
+Logs will be prefixed with `CELO_PNP_ERROR_XX`.  You can see a full list of them in [error.utils.ts](https://github.com/celo-org/celo-monorepo/blob/master/packages/phone-number-privacy/signer/src/common/error-utils.ts).
```
Comment (jmrossy) [inline packages/phone-number-privacy/signer/README.md]: Nit: Only error logs will

**Review by jmrossy — APPROVED** [review]
Looks great! One nit below

File: packages/phone-number-privacy/signer/README.md (hunk 3 above)
Comment (aslawson) [reply packages/phone-number-privacy/signer/README.md]: kk will update

File: packages/phone-number-privacy/signer/README.md (hunk 2 above)
Comment (aslawson) [reply packages/phone-number-privacy/signer/README.md]: thanks, I didn't know too much about these.  i'll update with what you've mentioned

**Pull request merged**